src/nci_adapter.c

```diff
@@ -67,6 +68,7 @@ typedef struct nci_adapter_intf_info {
     NCI_MODE mode;
     GUtilData mode_param;
     GUtilData activation_param;
+    NciModeParam* mode_param_parsed;                /* Since 1.0.11 */
```

Comment:
No need for these Since tags in the internal headers/sources. They only make sense for public interfaces, which other binaries can link against (to help those other binaries to keep their package requirements up-to-date).

Reply:
Thanks a lot for the review, comment was deleted